### ISSUE TITLE
The chat bot have unlabelled buttons and the screen reader doesn't recognize them.

### ISSUE THREAD

**Issue by sarperarikan** [open-webui/open-webui#4577]
The chat bot screen should have labelled buttons such as message send, Listen by TTS, Speech now. You can use aria-label techniques for them.
Best practice usage is;
<button type="button" accesskey="s" aria-label="Send a message">Send Message</button>

**Comment by tjbck** [comment]
#2790 PR welcome!
